src/ecrn_app.erl

```diff
@@ -22,9 +22,9 @@
 %% @doc
 %%  start up the app and all the dependent apps.
 manual_start() ->
-    application:start(crypto),
+    %application:start(crypto),
```

Comment:
Same as the warnings, what's your logic behind not starting crypto and sasl? Just want some clarification on your logic

Reply:
I wasn't sure why those two apps were needed in the first place. None of the test cases depend on them, and starting sasl might create unnecessary noise in the output.